src/components/Explore/ExploreChart.tsx

```diff
@@ -1,26 +1,82 @@
-import React, { FunctionComponent } from 'react';
+import React, { useCallback, Fragment } from 'react';
 import moment from 'moment';
-import useMediaQuery from '@material-ui/core/useMediaQuery';
-import { useTheme } from '@material-ui/core/styles';
-import { scaleTime, scaleLinear } from '@vx/scale';
+import { AxisLeft, AxisBottom } from '@vx/axis';
 import { GridRows, GridColumns } from '@vx/grid';
 import { Group } from '@vx/group';
-import { AxisLeft, AxisBottom } from '@vx/axis';
+import { scaleTime, scaleLinear } from '@vx/scale';
+import { useTooltip } from '@vx/tooltip';
+import { formatInteger } from 'common/utils';
 import { Column } from 'common/models/Projection';
 import * as ChartStyle from 'components/Charts/Charts.style';
-import RectClipGroup from 'components/Charts/RectClipGroup';
+import { Tooltip, RectClipGroup } from 'components/Charts';
 import { Series } from './interfaces';
-import SeriesChart from './SeriesChart';
-import { getMaxBy, getTimeAxisTicks } from './utils';
+import SeriesChart, { SeriesMarker } from './SeriesChart';
+import ChartOverlay from './ChartOverlay';
+import { getMaxBy, getTimeAxisTicks, findPointByDate } from './utils';
 import * as Styles from './Explore.style';
 
 const getDate = (d: Column) => new Date(d.x);
 const getY = (d: Column) => d.y;
+const daysBetween = (dateFrom: Date, dateTo: Date) =>
+  moment(dateTo).diff(dateFrom, 'days');
+
+const DateMarker: React.FC<{ left: number; date: Date }> = ({ left, date }) => (
+  <Styles.DateMarker style={{ left }}>
+    {moment(date).fromNow()}
+  </Styles.DateMarker>
+);
+
+const ExploreTooltip: React.FC<{
+  date: Date;
+  series: Series[];
+  left: (d: Column) => number;
+  top: (d: Column) => number;
+}> = ({ series, left, top, date }) => {
+  const [serieRaw, serieSmooth] = series;
+  const pointSmooth = findPointByDate(serieSmooth.data, date);
+  const pointRaw = findPointByDate(serieRaw.data, date);
+  return (
+    <Tooltip
+      top={top(pointSmooth)}
+      left={left(pointSmooth)}
+      title={moment(date).format('MMM D, YYYY')}
+    >
+      <Styles.TooltipSubtitle>{serieRaw.tooltipLabel}</Styles.TooltipSubtitle>
+      <Styles.TooltipMetric>{formatInteger(pointRaw.y)}</Styles.TooltipMetric>
+      <Styles.TooltipLocation>in NY</Styles.TooltipLocation>
+    </Tooltip>
+  );
+};
 
-const ExploreChart: FunctionComponent<{
+const DataMarkers: React.FC<{
```

Comment:
I guess this is what bolds the currently-hovered bar chart?

Would it make sense to name it DataHoverMarkers or something?  And/or give it a comment?